bodhi/server/services/updates.py

```diff
@@ -93,6 +94,14 @@
     cors_origins=bodhi.server.security.cors_origins_rw
 )
 
+update_get_test_results = Service(
+    name='update_get_test_results',
+    path='/updates/{id}/get-test-results',
+    description='Get test results for a specified update',
+    factory=security.PackagerACLFactory,
```

Comment:
Why do we want to limit this query to packagers?

Reply:
removed